TrafficReplayer/src/org/opensearch/migrations/replay/RequestResponseResponseTriple.java

```diff
@@ -0,0 +1,84 @@
+package org.opensearch.migrations.replay;
+
+import com.google.common.primitives.Bytes;
+import org.json.HTTP;
+import org.json.JSONObject;
+import org.opensearch.migrations.replay.TrafficReplayer.RequestResponsePacketPair;
+
+import java.io.BufferedOutputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.nio.charset.Charset;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Path;
+import java.time.Duration;
+import java.util.List;
+
+public class RequestResponseResponseTriple<write> {
```

Comment:
What is the generic param 'write' used for?

Reply:
Great question. No clue and no recollection of what I was thinking.